MediaManager/Plugin.MediaManager.Abstractions/IPlaybackManager.cs

```diff
@@ -49,6 +49,11 @@ public interface IPlaybackManager
         event MediaFinishedEventHandler MediaFinished;
 
         /// <summary>
+        /// Raised when media is finished playing.
```

Comment:
Please update this comment to have a better description when this event is thrown and why.